lib_nbgl/src/nbgl_use_case.c

```diff
@@ -302,6 +313,81 @@ static const nbgl_content_t *getContentAtIdx(const nbgl_genericContents_t *gener
     }
 }
 
+static void prepareNavInfo(bool isReview, uint8_t nbPages, const char *rejectText)
+{
+    memset(&navInfo, 0, sizeof(navInfo));
+
+    navInfo.nbPages           = nbPages;
+    navInfo.tuneId            = TUNE_TAP_CASUAL;
+    navInfo.progressIndicator = isReview;
+
+    if (isReview == false) {
+        navInfo.navType                   = NAV_WITH_BUTTONS;
+        navInfo.navWithButtons.navToken   = NAV_TOKEN;
+        navInfo.navWithButtons.backButton = true;
+    }
+    else {
+        navInfo.quitToken = REJECT_TOKEN;
+#ifdef TARGET_STAX
+        navInfo.navType                  = NAV_WITH_TAP;
+        navInfo.navWithTap.nextPageToken = NEXT_TOKEN;
+        navInfo.navWithTap.quitText      = rejectText;
+        navInfo.navWithTap.backToken     = BACK_TOKEN;
+#else   // TARGET_STAX
+        UNUSED(rejectText);
+        navInfo.navType                   = NAV_WITH_BUTTONS;
+        navInfo.navWithButtons.quitText   = "Reject";
+        navInfo.navWithButtons.navToken   = NAV_TOKEN;
+        navInfo.navWithButtons.backButton = true;
+#endif  // TARGET_STAX
+    }
+}
+
+static void prepareReviewFirstPage(nbgl_contentCenteredInfo_t *centeredInfo,
+                                   const nbgl_icon_details_t  *icon,
+                                   const char                 *reviewTitle,
+                                   const char                 *reviewSubTitle)
+{
+    centeredInfo->icon  = icon;
+    centeredInfo->text1 = reviewTitle;
+    centeredInfo->text2 = reviewSubTitle;
+#ifdef TARGET_STAX
+    centeredInfo->text3 = NULL;
+#else   // TARGET_STAX
+    centeredInfo->text3 = "Swipe to review";
```

Comment:
Not sure, but you seem to consider the `#else` for Europa? Is this function also available for Nano devices?